### PR TITLE
changes for edge bindings

### PR DESCRIPTION
**Pull request by yzainee**
Changes for the snyk vuln ingestion where edges were missing binding.

### CODE REVIEW

File: src/cve.py [hunk 1]
```diff
@@ -573,13 +578,13 @@ def put(payload):
 # add edge between CVE node and Version node if it does not exist previously
 add_affected_snyk_edge_script_template = """\
 cve_v=g.V().has('snyk_vuln_id',snyk_vuln_id).next();\
-version_v=g.V().has('pecosystem','{ecosystem}')\
-.has('pname','{name}')\
-.has('version','{version}');\
+version_v=g.V().has('pecosystem', ecosystem)\
+.has('pname', name)\
+.has('version', {version});\
```
Comment (arajkumar) [inline src/cve.py:583]: Still version properties is not using binding. Can you fix this?

Comment (yzainee) [reply src/cve.py:583]: Im not replacing it with absolute values. {version} is used to replace it with placeholders like version1, version2 and so on. Similarly the same placeholders are added in the bindings like {'version1': '1.2' ...}. Probably you got confused. Can you please re check

Comment (arajkumar) [reply src/cve.py:583]: To stop bloating gremlin groovy cache the query should be constant. Here I see that query is changing.

Comment (yzainee) [reply src/cve.py:583]: ok got it.. let me check on it

Comment (yzainee) [reply src/cve.py:583]: i got your point.. im making the changes and testing

File: src/cve.py [hunk 2]
```diff
@@ -203,10 +187,12 @@ def process(self):
                 else:
                     try:
                         # Connect CVE node with affected EPV nodes
-                        edge_queries, edge_bindings = self.get_qstrings_for_edges(vulnerability)
-                        for query_str in edge_queries:
+                        edge_query = add_affected_snyk_edge_script_template
+                        edge_bindings = self._get_default_bindings(vulnerability)
+                        for vuln_version in vulnerability.get('affected'):
+                            edge_bindings['vuln_version'] = vuln_version
```
Comment (arajkumar) [inline src/cve.py:193]: You can send `vuln_version` as a array type in the binding and iterate it the query. 
`ver.each { query}`

I've used this pattern in backbone, please check. https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/src/v2/stack_aggregator.py#L264

Comment (yzainee) [reply src/cve.py:193]: help me understand, how will this help or change anything?

Comment (yzainee) [reply src/cve.py:193]: @arajkumar can we close this conversation? PR merge is pending due to this.

Comment (arajkumar) [reply src/cve.py:193]: As discussed offline, you can still rollback the operation evening when using query side loop using http error codes.

**Pull request merged**